pom.xml

```diff
@@ -133,7 +133,7 @@
         <dependency>
   			<groupId>com.github.thebusybiscuit</groupId>
   			<artifactId>CS-CoreLib2</artifactId>
-  			<version>0.5.6</version>
+  			<version>f492af0f15</version>
```

Comment:
Sorry, only gonna do proper releases on that.

Reply:
REEEE